sensors/simsensor_common/simsensor_reader.c

```diff
@@ -0,0 +1,270 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Simsensor reader reads and parses data form special scenario file
+ * than it pushes it to event_queue
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Piotr Nieciecki
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <stdbool.h>
+
+#include "simsensor_reader.h"
+
+#define READER_END_SCENARIO_INDICATOR NO_SENSOR
+#define READER_DATA_SEPARATOR         ','
+
+#define READER_TIMESTAMP_NOT_SET -1
+
+
+static int reader_skipHeader(simsens_reader_t *rd)
+{
+	size_t lineLen = getline(&rd->lineBuf, &rd->bufLen, rd->scenarioFile);
```

Comment:
```suggestion
	ssize_t lineLen = getline(&rd->lineBuf, &rd->bufLen, rd->scenarioFile);
```
https://github.com/phoenix-rtos/libphoenix/blob/a514d66740f94232373d32ee7bb8d4ceef92ed9e/include/stdio.h#L360